src/saveload/saveload.h

```diff
@@ -304,6 +304,7 @@ enum SaveLoadVersion : uint16 {
 	SLV_TRADING_AGE,                        ///< 217  PR#7780 Configurable company trading age.
 	SLV_ENDING_YEAR,                        ///< 218  PR#7747 v1.10 Configurable ending year.
 	SLV_REMOVE_TOWN_CARGO_CACHE,            ///< 219  PR#8258 Remove town cargo acceptance and production caches.
+	SLV_INDUSTRY_TEXT,                      ///< 220  PR#???? Additional GS text for industries.
```

Comment:
This line should go after the line that says `SLV_GS_INDUSTRY_CONTROL` as the patchpack savegames are skipped for a valid reason.